.github/workflows/nightly-link-checker.yml

```diff
@@ -0,0 +1,62 @@
+name: Nightly Job
+
+# on:
+#  schedule:
+#    - cron: "1 5 * * *" # This schedule runs every night at 12:01 AM Eastern time (adjust according to your timezone)
+
+on: pull_request
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      NODE_OPTIONS: '--max_old_space_size=3584'
+
+    strategy:
+      matrix:
+        node-version: [14.x]
+
+    steps:
+      - name: Checkout repository!
+        uses: actions/checkout@v3
+      - name: Setup node - ${{ matrix.node-version }}
+        uses: actions/setup-node@v3
+        with:
+         node-version: ${{ matrix.node-version }}
+         cache: 'npm'
+      - name: Install dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Run script and capture logs  #broken link checker script
+        run: yarn build
+
+      - name: Run link checker #broken link checker script
+        run: yarn broken-link-checker:external > output.txt 2>&1 || true
+
+     # - name: Extract URLs from log
+        # run: |
+       #   cat output.txt | grep -Eo 'https?://[^ ]+' > urls.txt
+       # continue-on-error: true
+
+      - name: Extract URLs from log
+        run: |
+          grep -o 'Link: https\?://[^[:space:]]*' output.txt | sed 's/Link: //' > urls.txt
+        continue-on-error: true
+
+      - name: Remove False Positives
+        run: |
+          comm -23 <(sort urls.txt | uniq) <(sort falsepos.txt | uniq) > broken_external_urls.txt
+
+      - name: Read URLs from file
+        id: read-urls
```

Comment:
nit: let's remove `id` for now and will add it in the next PR when it will be required for slack action.